src/map/lua/lua_statuseffect.cpp

```diff
@@ -286,6 +286,14 @@ inline int32 CLuaStatusEffect::addMod(lua_State* L)
 
 //======================================================//
 
+inline int32 CLuaStatusEffect::getFlag(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PLuaStatusEffect == nullptr);
+
```

Comment:
I *think* we have a different method to do this, but I vote we keep what wiggo added here. It makes more sense / is more intuitive to do this way when I want to know wtf flag is set.

Reply:
The other way prolly being `somecrapStatusEffectByFlag()` if it exists. Teo fuzzy memory strikes again.

Reply:
yes, we do have hasStatusEffectByFlag and delStatusEffectsByFlag but nothing to return all the statuseffectsbyflag so that we can thumb through them for specifics (i.e. reraise in this example)
I tried going that route first.

Reply:
The other way I seen it being handled was to make a huge table of all pertinent effects and then cycle through the table like as in:
https://github.com/DarkstarProject/darkstar/blob/3ff2fefa3d1ffbdf2cd2cc3e3e03f85a9833b004/scripts/globals/abilities/benediction.lua#L15

Reply:
Yeah I like how you did it, was point of my comment. :+1: 

:smiley: